spring-cloud-task-core/src/main/java/org/springframework/cloud/task/configuration/SimpleTaskConfiguration.java

```diff
@@ -111,6 +112,7 @@ public TaskNameResolver taskNameResolver() {
 	}
 
 	@Bean
+	@ConditionalOnProperty(prefix = "spring.cloud.task", name = "tablePrefix", matchIfMissing = true)
```

Comment:
How does this prevent this from being initialized?  This looks like if the `spring.cloud.task.tablePrefix` is there, it will create the bean.  Can we get a test to prove this works?